generate-release-notes.js

```diff
@@ -28,11 +30,12 @@ async function generateReleaseNotes({
 	if (commits.length === 0) {
 		commitEntries.push('_Maintenance release_');
 	} else {
-		for (const {hash, title} of commits) {
+		for (const {hash, date, title} of commits) {
 			const line = commitTemplate
 				.replace('{hash}', hash)
 				.replace('{title}', title)
-				.replace('{url}', repoURL + '/commit/' + hash);
+				.replace('{url}', repoURL + '/commit/' + hash)
+				.replace('{date}', '[' + date + '](' + repoURL + '/commit/' + hash + ')');
```

Comment:
```suggestion
				.replace('{date}', date);
```

The link is unrelated. If the user wants a linked date they can use `[{date}]({url})` or whatever format they please

Reply:
OK, I add it as you wants. Also, I rewrite it in readme.